Documentation/Dev/Environment Setup.md

```diff
@@ -13,16 +13,18 @@ If either of these ports is taken by something already message the group and we
 4. Setup your MySQL Root Password. 
 5. Setup a dev user, this is the account we should be using to test our development setup.
 
-NOTE: Having Usernames and Password in github IS VERY INSECURE, DO NOT EXPOSE YOUR DB TO THE INTERNET. This username and password is something I made up just for this.
+Note: Having Usernames and Password in github IS VERY INSECURE, DO NOT EXPOSE YOUR DB TO THE INTERNET. This username and password is something I made up just for this.
 ```cmd
 User-type: DB Admin
 Username: notAdmin
 Password: notAdmin99!!
 ``` 
```

Comment:
Needs to be updated to be pulled from a local setup file and not hard-coded.

Reply:
Also this documentation should be updated when that happens to specify that any username and password are acceptable so long as the constants file is setup properly.